requirements.txt

```diff
@@ -1,6 +1,10 @@
+# top level requirements can be generated with
+# pipdeptree -fd 0
+
```

Comment:
Why are these lines here? I think the requirements file should be as clean as possible.

Reply:
I noticed requirements.txt only featured top-level dependencies, so I included a comment showing users how to generate a requirements.txt file in this style. Can be removed.

Reply:
Got it. My intention with requirements.txt is having the canonical dependencies. I know listing the dependencies and adding them to the requirement.txt file usually has so many dependencies it's hard to tell what are the "actual" dependencies.

Doesn't hurt to keep it in.